### PR TITLE
docs: Add quick start guide to README.md

### PR DESCRIPTION
**Pull request by shinfan**
- Add compatibility summary
- Add quick start guide
- Add configuration examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,114 @@
...
 
 To interact the client certificates, application code should not need to use most of these proxies within this repository directly. Instead, the application should leverage the clients and toolings provided by Google such as [Cloud SDK](https://cloud.google.com/sdk) to have a more convenient developer experience.
 
-## Installation
+## Compatibility
 
-TBD
+Currently ECP is in Preview stage and all the APIs and configurations are **subject to change**.
+
+Tthe following platforms are supported by ECP:
```
Comment (andyrzhao) [inline README.md:11]: Typo "The"

File: README.md [hunk 2]
```diff
@@ -1,12 +1,114 @@
...
+
+- MacOS (Keychain, PKCS#11)
```
Comment (andyrzhao) [inline README.md:13]: MacOS only supports Keychain by design and Windows only supports Window Certificate Store. To hypothetically support PKCS11 on MacOS would require compiling the Linux-targeted PKCS11 ECP code into a MacOS binary, and installing that on to the device (without gCloud bundling support.) Alternatively, we would need to augment the MacOS ECP to support pkcs11 in addition to keychain. Either way, there will be a lot of work and validation to be done in this area, so it's premature to document this. support.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,114 @@
...
+1. Enable usage of client certificates through gcloud CLI config command:
+```
+gcloud config set context_aware/use_client_certificate true
+```
+
+### Configuration
+
+ECP relies on the `certificate_config.json` file to read all the metadata information of locating the certificate. The contents of this JSON file looks like the following:
+
+#### MacOS (Keychain)
```
Comment (andyrzhao) [inline README.md:59]: Please update all the config examples based on Carl's config design doc. We hope to get the config change in before public preview.

**Review by andyrzhao — CHANGES_REQUESTED** [review]
Thanks for this PR! Please see comments.

File: README.md (hunk 3 above)
Comment (shinfan) [reply README.md:59]: Done.

**Comment by shinfan** [comment]
@andyrzhao PTAL

**Review by andyrzhao — APPROVED** [review]
LGTM.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,114 @@
...
+
+- MacOS: __Keychain__
+- Windows: __MyStore__
+- Linux: __PKCS#11__
+
+## Quick Start Guide
+
+### Prerequisites
+
+Before using ECP with your application/client, you should follow the instructions [here](enterprisecert) to configure your enterprise certificate policies with Access Context Manager.
```
Comment (sai-sunder-s) [inline README.md:21]: The link didn't work for me

File: README.md [hunk 5]
```diff
@@ -1,12 +1,114 @@
...
+
+## Quick Start Guide
+
+### Prerequisites
+
+Before using ECP with your application/client, you should follow the instructions [here](enterprisecert) to configure your enterprise certificate policies with Access Context Manager. 
+
+### Installation
+
+1. Install Openssl
```
Comment (sai-sunder-s) [inline README.md:25]: nit: fix numbering

**Pull request merged**